extension/apmconfigextension/internal/examples/agent/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
As `/bin/` starts with a `/` the absolute path of the root of the git repository is used. Is this intended?

Reply:
good catch, it should be relative https://github.com/elastic/opentelemetry-collector-components/pull/39/commits/62a96128d0620bec3b436d13775ba2082d344879